src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -27,12 +31,24 @@ const TopicDomainHeader = (props) => {
       <Pressable onPress={handlePress} style={{backgroundColor: 'transparent'}}>
         <View style={{flexDirection: 'row', alignItems: 'center'}}>
           <FastImage testID="imageTopicMember" source={TopicMemberIcon} style={styles.member} />
-          <Text style={styles.domainMember}>{memberCount} Members</Text>
+          {props?.initialData?.memberCount === undefined && props.isLoading ? (
+            <Shimmer height={10} width={normalize(25)} />
+          ) : (
+            <Text style={styles.domainMember}>
+              {props?.initialData?.memberCount || memberCount}
+            </Text>
+          )}
+          <Text style={styles.domainMember}> Members</Text>
         </View>
-        {isFollow && !hideSeeMember && (
-          <Text style={styles.seeMemberText} numberOfLines={1} ellipsizeMode="tail">
-            See community member
-          </Text>
+        {props?.initialData?.memberCount === undefined && props.isLoading ? (
+          <Shimmer height={10} width={normalize(60)} />
+        ) : (
+          isUserFollow &&
+          !hideSeeMember && (
+            <Text style={styles.seeMemberText} numberOfLines={1} ellipsizeMode="tail">
+              See community member
+            </Text>
+          )
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering logic here is quite complex and could be simplified for better readability. Consider extracting the conditions into well-named variables or helper functions.
For example, you could create a variable `isLoadingMemberCount` for the condition `props?.initialData?.memberCount === undefined && props.isLoading`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->